project_scrum/project_scrum.py

```diff
@@ -0,0 +1,500 @@
+# -*- coding: utf-8 -*-
+from openerp import models, fields, api, _
+from datetime import date, timedelta
+import re
+import logging
+try:
+    from bs4 import BeautifulSoup
+except ImportError:
+    logger = logging.getLogger(__name__)
+    logger.warning('bs4  are not available in the sys path')
+
+_logger = logging.getLogger(__name__)
+
+class scrum_sprint(models.Model):
+    _name = 'project.scrum.sprint'
+    _description = 'Project Scrum Sprint'
+    _order = 'date_start desc'
+
+    def get_current_sprint(self, project_id):
+        sprint = self.env['project.scrum.sprint'].search(['&', '&', 
+            ('date_start', '<=', date.today()),   
+            ('date_stop', '>=', date.today()),
+            ('project_id', '=', project_id)
+            ])
+        if len(sprint) > 0:
+            return sprint[0]
+        return None
```

Comment:
I'd suggest to return an empty browse record here - then code using this function can treat the result with less caution